marklogic-data-hub-central/ui/src/components/entities/mapping/source-entity-map/source-to-entity-map.test.tsx

```diff
@@ -1,89 +1,45 @@
-    });
 
-    test('RTL tests with source data',  () => {
-        const { getByTestId,  getByText,container, queryByText } = render(<SourceToEntityMap {...data.mapProps}  mappingVisible={true}/>);
-        expect(getByText('Source Data')).toBeInTheDocument();
-        expect(getByText('proteinId')).toBeInTheDocument();
-        expect(getByTestId("entityContainer")).toBeInTheDocument();
-        expect(getByTestId("srcContainer")).toBeInTheDocument();
-        expect(getByTestId("srcContainer")).toHaveClass("sourceContainer");
-        expect(getByText('Entity: Person')).toBeInTheDocument();
-        expect(getByText('Test')).toBeEnabled();
-        expect(queryByText("Unable to find source documents using the specified collection or query.")).not.toBeInTheDocument();
-        let exp = getByText('mappedName');
-        expect(exp).toBeInTheDocument();
-        fireEvent.change(exp, { target: {value: "concat(name,'-NEW')" }});
-        fireEvent.blur(exp);
-        fireEvent.click(getByText('Clear'));
-        expect(getByText('Clear')).toBeEnabled();
-        expect(getByText("concat(name,'-NEW')")).toBeInTheDocument();
-        console.log(container);
-    });
-});
-
-describe('Enzyme Source-to-entity map tests', () => {
-    let wrapper: any;
-    beforeEach(() => {
-        wrapper = shallow(<SourceToEntityMap {...data.mapProps} />);
-    });
+  afterEach(cleanup);
 
-    test('Enzyme tests with source data', () => {
-        //Use console.log(wrapper.debug()) for debugging the html returned by the wrapper;
-        expect(wrapper.find('#srcContainer').length).toEqual(1);
-        expect(wrapper.find('#srcDetails').length).toEqual(1);
-        expect(wrapper.find('#entityContainer').length).toEqual(1);
-        expect(wrapper.find('#noData').length).toEqual(0);
-        expect(wrapper.find('#dataPresent').length).toEqual(1);
-        //Success and Error message are shown only when a mapping expression is being saved
-        expect(wrapper.find('#successMessage').length).toEqual(0);
-        expect(wrapper.find('#errorMessage').length).toEqual(0);
-        //List and Function icon are displayed only when the entity table loads with entity properties
-        expect(wrapper.find('#listIcon').length).toEqual(0);
-        expect(wrapper.find('#functionIcon').length).toEqual(0);
-        expect(wrapper.find('#Clear-btn').length).toEqual(1);
-        expect(wrapper.find('#Test-btn').length).toEqual(1);
-        expect(wrapper.find('#errorInExp').length).toEqual(0);
-        expect(wrapper.find('#valuesAfterTest').length).toEqual(0);
-        const splitPane = wrapper.find(SplitPane);
-        expect(splitPane).toHaveLength(1);
-        expect(splitPane.prop('split')).toEqual('vertical');
-        expect(splitPane.prop('primary')).toEqual('second');
-        expect(splitPane.prop('allowResize')).toEqual(true);
-        expect(wrapper.find(SplitPane).at(0).find('#srcContainer').length).toEqual(1);
```

Comment:
Why are these enzyme tests being removed ? I couldn't test splitpane with RTL,  so I added enzyme test for it

Reply:
Those tests are there still and are not removed. Git diff is confusing